src/Compilers/VisualBasic/Test/Syntax/Syntax/SyntaxSubmissionsTests.vb

```diff
@@ -0,0 +1,116 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class SyntaxSubmissionsTests
+        Public Sub AssertCompleteSubmission(code As String)
```

Comment:
I'm having some trouble viewing the diff across revisions of this PR.  Was this ever addressed?